### PR TITLE
Added umutarray page

### PR DESCRIPTION
**Pull request by umutarray**
## Description

Here is my personal page 

,
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

Your ENS/address: 0x3566ce434c0d8B80934354c806f8651e1cC91EDD

### CODE REVIEW

File: packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Image from "next/image";
+
+const umutarray = () => {
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx:4]: Consider renaming the component to Umutarray to follow React's PascalCase convention.
Also, it's recommended to use the NextPage type from the next package for defining page components in Next.js.

File: packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            </div>
+            <h1 className="text-5xl font-bold">I am Umutarray</h1>
+            <p className="py-6 text-xl">
+              I am blockhain enthusiast second-year computer science student and i have been in crypto since 2020.
+            </p>
+
+            <p className="py-6 text-xl">you are always welcome to reach out to me via my social accounts.</p>
+
+            <a className="link link-primary" href="https://x.com/umutarray" target="_blank">
+              <button className="btn bg-pink-300">Twitter</button>
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx:23]: You could improve the design by incorporating symbols or emojis to make it more visually appealing (for Twitter, Github, BuidlGuidl).

**Review by phipsae — CHANGES_REQUESTED** [review]
Thanks for the PR @umutarray!

As a general advice, please ensure **you create and check out a new branch** (see contribution guideline, https://gist.github.com/ZakGriffith/69d1eb8baebddd7d370b87a65a7e3ec0), so that you adhere to the guidelines, which is important to contribute to open source projects.

Below, you'll find a few comments on minor code improvements.

File: packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx (hunk 2 above)
Comment (umutarray) [reply packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx:24]: thanks for your advice but i liked that design more

**Comment by umutarray** [comment]
@phipsae I editted my code. can you check it please ?

**Comment by derrekcoleman** [comment]
Hey @umutarray! I love the simple color palette you're using here. My eyes can deal with the low contrast between pink and white, but others' may not be able to. Between you and me, it's gorgeous!

Please make sure you're using [Prettier](https://prettier.io/) when saving so we all have consistent conventions (e.g. semicolons or no) and to fix the build errors we're seeing at the moment:
> ./app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx
5:7  Error: 'umutArray' is assigned a value but never used.  @typescript-eslint/no-unused-vars
5:16  Warning: Delete `·`  prettier/prettier

Even that first one about `umutArray` will go away after you use Prettier. I believe it's being triggered because of the space between `umutArray :` and the colon `:` in line five. Try it out and lmk if you have any questions.

I also recommend you add some padding between your social media buttons.

**Comment by umutarray** [comment]
@derrekcoleman can you check it please :)

File: packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+
+const umutArray:NextPage = () => {
```
Comment (derrekcoleman) [inline packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx:5]: There's still one pesky Lint error related to a missing symbol on this line (careful, that's not a period it's talking about)
```
./app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx
5:17  Warning: Insert `·`  prettier/prettier
```

Make sure you run `yarn build` (while your terminal is in the `nextjs` folder) so you can catch these little errors _before_ you push commits.

**Review by derrekcoleman — CHANGES_REQUESTED** [review]
@umutarray thanks for the fixes and the padding 👌 

See my requested change for another Prettier error that's making CI fail.

**Comment by umutarray** [comment]
Hi @derrekcoleman,
I made it fixed. Can you please look at ?

**Comment by derrekcoleman** [comment]
Thank you for your persistence with the build errors! Merged 🚀

**Pull request merged**